apps/rush-lib/src/cli/actions/PublishAction.ts

```diff
@@ -390,12 +385,16 @@ export class PublishAction extends BaseRushAction {
 
   private _packageExists(packageConfig: RushConfigurationProject): boolean {
     const env: { [key: string]: string | undefined } = PublishUtilities.getEnvArgs();
-    if (this._registryUrl.value) {
-      env['npm_config_registry'] = this._registryUrl.value; // tslint:disable-line:no-string-literal
+    const registry: string = this._getRegistryAndUpdateEnv(env);
+    let args: string = '';
```

Comment:
>string [](start = 14, length = 6)

Weird that this is a `string` and the other one is a `string[]`. Can we make these consistent?